src/ui/components/Scanner/Scanner.tsx

```diff
@@ -295,6 +293,53 @@ const Scanner = forwardRef(
           return;
         }
 
+        throw e;
+      }
+    };
+
+    const resolveConnectionOobi = async (content: string) => {
+      // Create pending connection item on UI. This will be remove after create connection process end.
```

Comment:
I think this comment can stay, please change it to:
```
// Adding a pending connection item to the UI. 
// This will be removed when the create connection process ends.
```

Reply:
Done